src/content/docs/apm/agents/python-agent/configuration/python-agent-configuration.mdx

```diff
@@ -3399,7 +3531,9 @@ Event harvest configuration settings include:
       </tbody>
     </table>
 
-    Limit for custom events per minute sent by an instance of the Python agent to New Relic. Custom events are created through the Python Telemetry SDK.
+    * Limits how many custom events per minute that an instance of the Python agent can send to New Relic. 
+    * Custom events are created through the Python Telemetry SDK.
```

Comment:
This is just outdated and not correct at all so let's just remove this while we're in here.
```suggestion
```